lib/jekyll-import/importers/pebble.rb

```diff
@@ -15,7 +15,7 @@ def self.specify_options(c)
       end
 
       def self.process(opts)
-        options = { directory => opts.fetch("directory", "") }
+        options = { :directory => opts.fetch("directory", "") }
```

Comment:
Great work here! I saw that too (when I looked again). The key needs to be a symbol (used a colon `:`).